ui/src/components/ReleaseCard/index.jsx

```diff
@@ -166,6 +166,30 @@ function ReleaseCard(props) {
     return `/rules${qs}#ruleId=${ruleId}`;
   };
 
+  const highlightMatchedRelease = highlights => {
```

Comment:
**todo:** I would like a unit test for this `highlightMatchedRelease` function in case we have to change it in the future. You could move the function into a logic module like [elementsHeight](https://github.com/mozilla-releng/balrog/blob/main/ui/src/utils/elementsHeight.js). Then you can add a unit test in [ui/test](https://github.com/mozilla-releng/balrog/blob/main/ui/test). That will run as part of the [application's CI process](https://github.com/mozilla-releng/balrog/pull/3033/checks?check_run_id=18011913985). You can run tests locally using docker:
```
(balrog) ➜  balrog git:(highlight-partial-match-releases) docker exec -it balrog-balrogui-1 yarn test
yarn run v1.22.19
$ jest
 PASS  test/ListRules_test.jsx

Test Suites: 1 passed, 1 total
Tests:       10 passed, 10 total
Snapshots:   0 total
Time:        0.506 s, estimated 1 s
Ran all test suites.
Done in 0.98s.
```